ankihub/ankihub_client/ankihub_client.py

```diff
@@ -1270,10 +1270,6 @@ def send_daily_card_review_summaries(self, daily_card_review_summaries: List[Dai
         if response.status_code != 201:
             raise AnkiHubHTTPError(response)
 
-    def has_reviewer_extension_access(self) -> bool:
```

Comment:
This won't be necessary anymore, because we can use `check_user_feature_access()` instead.